rasa/core/channels/rasa_chat.py

```diff
@@ -73,30 +72,20 @@ async def _fetch_public_key(self) -> None:
                         "".format(public_key_url, public_key_field, json.dumps(rjs))
                     )
 
-    def _decode_jwt(self, bearer_token: Text) -> Dict:
-        if self.jwt_key is None:
-            raise ChannelConfigError(
-                "JWT public key is `None`. This is likely caused "
-                "by an error when retrieving the public key from Rasa X."
-            )
-
-        authorization_header_value = bearer_token.replace(
-            constants.BEARER_TOKEN_PREFIX, ""
-        )
-        return jwt.decode(
-            authorization_header_value, self.jwt_key, algorithms=self.jwt_algorithm
-        )
-
     async def _decode_bearer_token(self, bearer_token: Text) -> Optional[Dict]:
         if self.jwt_key is None:
             await self._fetch_public_key()
 
         try:
-            return self._decode_jwt(bearer_token)
+            return rasa.core.channels.channel._decode_jwt(
```

Comment:
it's still not public 🙈

Reply:
Ugh yeah I was blanking, I need a re-fresher on the engineering guidelines🤦